### PR TITLE
EOS-19790: Add initial node health during 'config' command

### PR DESCRIPTION
**Pull request by indrajitzagade**
## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    EOS-19790: Cluster manager get node health API
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Need to add initial node health till system health bootstrap has been implemented.
  </code>
</pre>
## Solution
<pre>
  <code>
    Updated mini-provisioner 'config' command to add initial node status as 'online'
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Ran mini-provisioner 'config' command on single node and confirmed health in the consul.
  </code>
</pre>

### CODE REVIEW

File: ha/setup/ha_setup.py [hunk 1]
```diff
@@ -570,21 +572,42 @@ def _update_node_map(self) -> None:
...
+                        NODE_MAP_ATTRIBUTES.RACK_ID.value: rack_id, NODE_MAP_ATTRIBUTES.STORAGESET_ID.value: storageset_id}
+            system_health = SystemHealth(self._confstore)
+            key = system_health._prepare_key(const.COMPONENTS.NODE_MAP.value, node_id=node_id)
+            # Check key is already exist if not, store the node map.
+            node_map_val = self._confstore.get(key)
+            if node_map_val is None:
+                self._confstore.set(key, str(node_map))
+
+            # TBD: Temporary code till EOS-17892 is implemented
+            initial_health = EntityHealth()
```
Comment (ajaysrivas) [inline ha/setup/ha_setup.py:592]: Please put this in different function, a private one.

**Review by ajaysrivas — CHANGES_REQUESTED** [review]

File: ha/setup/ha_setup.py (hunk 1 above)
Comment (indrajitzagade) [reply ha/setup/ha_setup.py:592]: @ajaysrivas  It is a temporary code which will be deleted soon. I needed some info e.g. cluster_id, site_id, etc. which is available already in this method, hence added it here. I can move, but is it really needed?

Comment (ajaysrivas) [reply ha/setup/ha_setup.py:592]: yes @indrajitzagade. If this code is separate, disabling/removing the code will be better. There will be less probability in making any error.

Comment (indrajitzagade) [reply ha/setup/ha_setup.py:604]: Done @ajaysrivas

**Review by ajaysrivas — APPROVED** [review]

**Review by ArchanaLimaye — APPROVED** [review]

File: ha/core/cluster/cluster_manager.py [hunk 2]
```diff
@@ -433,7 +433,7 @@ def is_status_present(self, element, element_id: str = HEALTH_STATUSES.UNKNOWN.v
                     break
         else:
             for key in self._status_dict:
-                if re.search(f"{element}/.+\w/health", key):
+                if re.search(f"{element}/.+/health", key):
```
Comment (indrajitzagade) [inline ha/core/cluster/cluster_manager.py:436]: @ajaysrivas @ArchanaLimaye I saw this issue on the setup where node ids are single digit e.g. '0', then that node status was not returned. I guess this "\w" was not required, hence removed. With this change the node status is returned fine.

**Pull request merged**